### DISCUSSION TITLE
What's the difference between branches "master" and "main"?

### DISCUSSION THREAD

**Discussion by rusq** [charmbracelet/bubbletea#1188] [Q&A] [answered]
Hey cheery peeps,

I can see that master branch is behind main by 21 commits, but at the same time ahead by 92?

In the "master" branch, the "KeyMsg" is deprecated in favour of "KeyPressMsg" and "KeyReleaseMsg".
In the "main" branch, the "KeyMsg" is still a thing.

When I "go get" the master branch it is v0.27 tag descendant, but main branch is v1.1.1+.

I'm just trying to understand the direction, as I quite like this type of Key press message segregation (brings good memories of MSDN and Win32API... no, kidding).  I'm asking out of curiosity, no particular problem, otherwise I would submit the bug report without blinking 😂

Thank you.

**Comment by aymanbagabas** [comment, marked as answer]
Hey @rusq, we're working towards a Bubble Tea V2 with lots of new features and changes. You can `go get github.com/charmbracelet/bubbletea/v2@v2-exp` to try it out. Read the [v2.0.0-alpha.1](https://github.com/charmbracelet/bubbletea/releases/tag/v2.0.0-alpha.1) release notes to get a glimpse of what we're working on.

In terms of the branches, we recently switched the default branch from `master` to `main`. Internally, we wanted to accelerate the development of v2 and merge things into `master` right away. But quickly we figured that would be an issue as well as going from v0.27 -> v2.0 without a proper v1 release. We then decided it would be a good time to switch the default branch and restore the commit history 😄

**Reply by rusq** [reply]
Hey @aymanbagabas, thank you for explanation!, will play around with v2 today, looking forward for the new features :)

Thanks for amazing library!
